scripts/pre-commit.bash

```diff
@@ -0,0 +1,22 @@
+# Flutter formatter
+printf "\e[33;1m%s\e[0m\n" '=== Running Flutter Formatter ==='
+
+hasNewFilesFormatted=$(git diff --name-only)
+flutter format $hasNewFilesFormatted
+if [ -n "$hasNewFilesFormatted" ]; then
+    git add .
+    printf "\e[33;1m%s\e[0m\n" 'Formmated files added to git stage'
+fi
+printf "\e[33;1m%s\e[0m\n" 'Finished running Flutter Formatter'
+printf '%s\n' "${avar}"
+
+# Flutter Analyzer
+printf "\e[33;1m%s\e[0m\n" '=== Running Flutter analyzer ==='
+flutter analyze
```

Comment:
Is it possible to run this only on modified files? 🤔

Reply:
It doesn't give -m files. thought will do it as part of linting ticket if required will remove this code snippet for now.